modules/etcd-overview.adoc

```diff
@@ -0,0 +1,32 @@
+// Module included in the following assemblies:
+//
+// * architecture/control-plane.adoc
+
+
+:_content-type: CONCEPT
+[id="etcd-overview_{context}"]
+= Overview of etcd
+
+etcd is a consistent, distributed key-value store that holds small amounts of data that can fit entirely in memory. Although etcd is a core component of many projects, it is the primary data store for Kubernetes, which is the standard system for container orchestration.
+
+[id="etcd-benefits_{context}"]
+== Benefits of using etcd
+
+By using etcd, you can enjoy several benefits:
```

Comment:
enjoy? 
_Using etcd can deliver several benefits_ ??

Reply:
You're right -- "enjoy" is kinda weird for technical docs. I like the spirit of your revision, but the IBM Style Word Usage page lists "using" as a term to use with caution (as opposed to "by using" or "that uses") because it can be ambiguous. This one is tricky because I don't want to start too many sentences with lowercase "etcd".

I can change it to something like, "By using etcd, you gain several benefits:" or "By using etcd, you can benefit in several ways". What do you think?

Reply:
"By using etcd, you can benefit in several ways" gets my vote!